api/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/gravitational/teleport/api
+
+go 1.15
+
+require (
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.3
+	github.com/gravitational/trace v1.1.14
+	github.com/jonboulle/clockwork v0.2.2 // indirect
+	github.com/sirupsen/logrus v1.7.0 // indirect
```

Comment:
I know it's indirect, but do you know what's still using `clockwork` and `logrus`?

Reply:
Yes, it's from `gravitational/trace`